Jenkinsfile

```diff
@@ -72,6 +72,17 @@ node {
           rsync -arv -e \"ssh -2\" ./chrome/*.json sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}/chrome
         """
       }
+
+      // set akamai fast purge env
+      openShiftUtils.withJnlpNode(image: "quay.io/redhatqe/jenkins-slave-python36-akamai:v3.11") {
```

Comment:
So that it's clear in the pipeline stage view, I think it would be good to put this in a separate stage (if possible).